pkg/js/parse_tests/014-caa.json

```diff
@@ -0,0 +1,48 @@
+{
+    "registrars": [],
+    "dns_providers": [],
+    "domains": [
+        {
+            "name": "foo.com",
+            "registrar": "none",
+            "dnsProviders": {},
+            "records": [
+                {
+                    "type": "CAA",
+                    "name": "@",
+                    "caatag": "issue",
+                    "caaflag": 0,
+                    "target": "letsencrypt.org"
+                },
+                {
+                    "type": "CAA",
+                    "name": "@",
+                    "caatag": "issuewild",
+                    "caaflag": 0,

```

Comment:
Since caaflags is "omitempty", I"m not sure this `"caaflag": 0,`  appears here.

Ah, I see the problem. The actual tag is "caaflags" (with an "s").  How about making it "caaflag" everywhere to match the dns.CAA struct?